user/articles/3-layout-and-design/12-widget-reference.markdown

```diff
@@ -52,99 +41,77 @@ image is assigned to the category in (set in *Site Menu* &rarr;
 If you place a Search Results widget on the same page, Category Content will
 point the search widget toward products in the selected category.
 
-![Figure 4: Category Content displays the heading for the category and populates the Search Results widget beneath it.](../../images/category-content.png)
-
 ## Checkout
 
 The checkout widget handles purchase process. Without any configuration, the
-*Checkout* widget features a four-step process: users are prompted to enter a
-shipping address, a billing address, an order summary, and finally an order
-confirmation. When you configure [Shipping
-Methods](/web/liferay-emporio/documentation/-/knowledge_base/1-0/shipping-methods)
-and [Payment
-Methods](/web/liferay-emporio/documentation/-/knowledge_base/1-0/payment-methods),
-additional steps are added automatically to the checkout portlet's process,
-prompting users to select one of each.
+Checkout widget features a four-step process: shipping address, billing address,
+order summary and order confirmation. When you configure 
+[Shipping Methods](/web/liferay-emporio/documentation/-/knowledge_base/1-0/shipping-methods)
+and 
+[Payment Methods](/web/liferay-emporio/documentation/-/knowledge_base/1-0/payment-methods),
+additional steps are automatically added to the process.
 
-![Figure 5: Checkout offers users a familiar experience out of the box.](../../images/checkout.png)
-
-Additional checkout steps can be customized. Details 
-[here](/web/liferay-emporio/documentation/-/knowledge_base/1-0/custom-checkout-step).
+Custom checkout steps can be added using the 
+[custom checkout step extension point](/web/liferay-emporio/documentation/-/knowledge_base/1-0/custom-checkout-step).
 
 ## Commerce Categories Navigation
```

Comment:
The way we talk about the Commerce Categories Navigation and Category Content widgets needs to be modified. It is very difficult to follow for a reader. We either need to go into much more detail (e.g. introducing the concept of friendly URLs and display pages) or make it very simplistic and avoid the actual inter-workings.

Reply:
Naturally, I prefer the latter for this particular doc.